config/smart_answers/check_travel_during_coronavirus_summary_text.yml

```diff
@@ -0,0 +1,8 @@
+---
+summary_text:
+  young_person: You are travelling with someone aged 17 and under
+  zero_to_four: You are travelling with someone aged 4 and under
+  five_to_seventeen: You are travelling with someone aged 5 to 17
+  fully_vaxed: You are fully vaccinated or cannot have a COVID-19 vaccination
+  not_vaxed: You are not fully vaccinated
+  red_list: You will be in a red list country within 10 days of returning to England
```

Comment:
I think the ordering is not quite right. In the previous world the content in `data[:reasons]` was displayed, first, so I think the order should be:

- fully_vaxed
- not_vaxed
- red_list
- young_person
- zero_to_four
- five_to_seventeen

This will ensure the order of the reasons displayed to the user remains unchanged.

Reply:
yeah that was silly, have fixed this

Reply:
Thinking about this morning, I think the order changing is due to the order of the conditionals in the `summary_text_fields` method rather than in this file. Oops!

Reply:
Also, I think this is a locale file isn't it? Perhaps it should be in `config/locales`? If you did that you wouldn't need a method to load the YAML file because Rails would do it for you.

Reply:
Have sorted this. Added a locale file with proper file structure and fixed the order that things are added to the `summary_text_fields` array that I've now renamed and stopped being an instance variable